### DISCUSSION TITLE
Re-scanning complete Tumblr

### DISCUSSION THREAD

**Discussion by stuwm** [TumblThreeApp/TumblThree#317] [Q&A (Question & Answer)]
I have been current with the download of several hidden Tumblrs. After a new login (not sure whether this is the cause) it is COMPLETELY re-scanning all hidden Tumblrs. Not sure how to reproduce, but it happened in the past and was gone some day.

**Expected behavior**
Scanning only for updates since the last successful download, not the complete Tumblr.

**Desktop (please complete the following information):**
 - TumblThree version: 2.4.7.40
 - OS: Windows 10 Professional
 - Browser Edge
 - Version 97.0.1072.55

**Comment by dajotim937** [comment]
Is 'force rescan' checkbox checked? In app settings or blog settings.

**Comment by stuwm** [comment]
In the application I didn't find that option, but in the blog settings it is NOT checked. I run the app in "portable mode", does that trigger anything?

**Comment by dajotim937** [comment]
Portable mode shouldn't trigger anything. But Force rescan should do thing that you talk about. Don't know any other reasons for app to rescan blogs.

<details>
<summary>Screenshots</summary>

![2022_01_13 21-13-44](https://user-images.githubusercontent.com/96221151/149394521-eae6a1f4-b630-4c02-a7d9-8e317cc56c42.png)
![2022_01_13 21-13-59](https://user-images.githubusercontent.com/96221151/149394526-163a0e83-4285-4101-a138-cffeaf476341.png)
</details>

**Comment by thomas694** [comment]
@stuwm
Login is only a precondition, but not affecting the way or result of scanning. Do you have the problem with all your hidden blogs or only some? Is there a recent (last few weeks) post in the affected blogs or are they older?
When you experienced it in the past, did it disappear after you updated to a new version or within the same version?
What's the `LastId` value in the blog files, 0 or some big number?
When you start the app and download one of your affected blogs, can you see an error in TumblThree.log (should be in the local folder for portable mode)?

**Comment by thomas694** [comment]
The problem is that the blog's post list isn't in chronological order when the blog owner uses a pinned post. The id from the pinned post was saved as last id, always crawling until that id again. The id of the first non-pinned post is saved now.

It's fixed in the next version.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
